examples/eth_provider/test/eth_provider_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule EthProviderTest do
+  use ExUnit.Case
+  doctest EthProvider
+
+  test "greets the world" do
+    assert EthProvider.hello() == :world
+  end
```

Comment:
I wonder how this test passes if we don't have an `hello/0` function

Reply:
If you wanna play with Git actions, you could have another action that would run only if for examples folder

Reply:
The best is to have a separate action that runs only for the examples.